server/src/index.ts

```diff
@@ -1,2 +1,119 @@
-const message = 'Hello World!';
-console.log(message);
+import { Challenge, Status } from './types/interfaces';
+import { startNewChallenge } from './services/startNewChallenge';
+import { getAchievements } from './services/getAchievements';
+import { getTaskArchive } from './services/getTaskArchive';
+import { calculateAchievementsStatus } from './services/calculateAchievementsStatus';
+import { tasks } from './mockDB/tasks.json';
+import { allAchievements } from './mockDB/achivements.json';
+import { generateActualAchievementsList } from './services/helpers/generateActualAchievementsList';
+
+import express = require('express');
+const config = require('config');
+
+// const cors = require('cors');
+
+const app = express();
+const server = require('http').createServer(app);
+const io = require('socket.io')(server);
+
+const PORT = config.get('port') || 5000;
+
+export const challenges: any = [];
+const achievementsList = generateActualAchievementsList(allAchievements);
+const tempChallenge: Challenge[] = [
```

Comment:
Why would you create a temp challenge instead of starting one handling a request to a certain route?